pkg/placement/placement.go

```diff
@@ -25,42 +24,99 @@ import (
 
 var log = logger.NewLogger("dapr.placement")
 
+type placementGRPCStream placementv1pb.Placement_ReportDaprStatusServer
+
+const (
+	// membershipChangeChSize is the channel size of membership change request from Dapr runtime.
+	// MembershipChangeWorker will process actor host member change request.
+	membershipChangeChSize = 100
+
+	// faultyHostDetectMaxDuration is the maximum duration when existing host is marked as faulty.
+	// Dapr runtime sends heartbeat every 1 second. Whenever placement server gets the heartbeat,
+	// it updates the last heartbeat time in UpdateAt of the FSM state. If Now - UpdatedAt exceeds
+	// faultyHostDetectMaxDuration, MembershipChangeWorker tries to remove faulty dapr runtime from
+	// membership.
+	faultyHostDetectMaxDuration = 3 * time.Second
```

Comment:
not needed for this PR possibly, but can all the times be made as configurable parameters?

Reply:
may be later.